[inductor] include global cache dir in inductor resources

nmacchioni opened the pull request:
Summary: adding global cache dir glob to inductor resources

Test Plan: sandcastle + CI + tested locally

Differential Revision: D46131451



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

jansel commented on torch/_inductor/config.py:
```diff
@@ -148,7 +148,10 @@ def decide_compile_threads():
-    global_cache_dir = "fb/cache"
+    from libfb.py import parutil
+
+    relative_path = os.path.join(__package__.replace(".", os.sep), "fb/cache")
+    global_cache_dir = parutil.get_dir_path(relative_path)
```
```suggestion
    global_cache_dir = parutil.get_dir_path(os.path.join(__package__.replace(".", os.sep), "fb/cache"))
```

So we don't create an extra config nameed `relative_path`

nmacchioni committed:
[inductor] include global cache dir in inductor resources (#102130)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/102130

adding global cache dir glob to inductor resources

Test Plan: sandcastle + CI + tested locally

Reviewed By: bertmaher

Differential Revision: D46131451

fbshipit-source-id: f4a314e8f1e52de274d90074a754afc4235d3840
